packages/zone/src/once.js

```diff
@@ -0,0 +1,29 @@
+/** @typedef {import('@endo/patterns').Passable} Passable */
+
+/**
+ * The best way to understand the purpose of `once` is to first understand
+ * what `once` does on a durable zone. Used correctly, `once` should only
+ * be called at most once on any subzone,key pair during any vat incarnation.
+ * Given that constraint, if there is already a value bound to that
+ * subzone,key pair, it must have been left there by a previous incarnation and
+ * `once` will simply return it. If not, then `makeValue(key)` is called to
+ * determine the initial value of that slot, which will normally be preserved
+ * by similar calls to `once` in future incarnations --- though that will be
+ * up to them.
+ *
+ * For single incarnation zones, `zone.once(key, makeValue)` just calls
+ * `makeValue(key)` and returns the result.
+ *
+ * Default zones do no uniqueness checking. It is up to the caller not to call
+ * `once` more than once per subzone,key pair within any vat incarnation.
```

Comment:
Why not perform uniqueness checks for non-durable zones?

Reply:
@michaelfig has a plan for that.

@michaelfig my inclination is to put this PR into Draft and wait for your pervasively uniqueness checking zone. Or even close this one in favor of your's. Does that seem plausible to you?

Reply:
> Or even close this one in favor of yours.
That's plausible.  Before that, I wanted to understand if you are also in favour of the rename I like inspired by @gibson042 -- `zone.makeOnce(key, maker)`.

Reply:
Yes. Go for it.